src/utils/requests/apolloRequests.js

```diff
@@ -12,12 +14,34 @@ const client = new ApolloClient({
   }
 });
 
+export function dispatchError(error) {
+  const { message } = error.graphQLErrors[0];
+  store.dispatch(setError(message));
+  store.dispatch(setFetching(false));
```

Comment:
Cool! So this is how we can have access to Redux with our queries?

Reply:
Yup!